### DISCUSSION TITLE
Megane 3 ph1 - Pulse turn signals

### DISCUSSION THREAD

**Discussion by LemurMort** [cedricp/ddt4all#1273] [Q&A]
Hello!
It's been a while since I was in 'ddt/pyren' sphere.
Recently I got a ph1 Megane 3 and I want my flashers to flash 3 times when I pull the lever into that position.
In ddt4all during scan a BCM block SW10 appears but in it there's no option to activate this option. But there is one in SW25 v1.1. I think it's called something like TP_PULSE. 

So, if I manually select SW25 and activate that option will I mess something up irreversibly?

Thanks!

**Comment by AndyIreland** [comment]
well I tell you what I would do, and I dont expect you to be like me , so choose at your own peril but

If i saw the setting TP_PULSE and on the 'read' side it showed enabled or disabled (or 0 or 1) then I would throw caution to the wind and try changing it. - if in the 'read' box it was coloured Red and had No Data in the box then I would not even attempt to change the settings. 

But , you know - work away and dont blame me if something goes wrong LOL

**Reply by LemurMort** [reply]
Thanks Andy, I appreciate your input as always. I'm curious to see if there's more different opinions on this one. 🙂

**Comment by LemurMort** [comment]
So, this is the current situation. I haven't dared to change anything.

After scanning with magnifying glass this BCM is discovered and under 'Flasher' this is what I have:
![Screenshot 2024-10-19 110943](https://github.com/user-attachments/assets/6a5e307a-99a5-41f5-b2aa-ad27f021f6da)

When I manually select BCM SW25 there's more options:
![Screenshot 2024-10-19 110842](https://github.com/user-attachments/assets/3ca1b8bc-b8f2-40e1-80f4-66140e38ed4e)

Within this SW25 after I go to 'Calibration' all is marked red no matter if I pull and release the lever or hold it in place. Nothing changes as the fields remain red.
![Screenshot 2024-10-19 111100](https://github.com/user-attachments/assets/b61a7082-0de6-4a6b-9ab0-56e6a7f762cf)

What do you guys think?
Thanks!

**Comment by AndyIreland** [comment]
@LemurMort - try increasing CAN timeout at the bottom there to 600ms and then press the bigger green button at the top there to refresh it - sometimes it can change the red box with 'No Data' . 
If that does not work I wouldnt change any parameter that has the red box there . 

There is manual way with 
![image](https://github.com/user-attachments/assets/16c72313-2746-4fb3-95bc-d774ad2d710d)
But again you have to be careful you could muck up your ECU

![image](https://github.com/user-attachments/assets/7b3f9987-0254-4e0c-810f-c8feaf12b48d)
